app/assets/stylesheets/components/_game-card.scss

```diff
@@ -0,0 +1,98 @@
+.game-card{
+  @media (max-width: 768px) {
+    display: none,
+  }
+
+    margin : auto;
+    margin-bottom: 3rem;
+    max-width: 800px;
+    max-height: 220px;
+    display: flex;
+
+
+  .picture{
+    height: 200px;
+    width : 315px;
+  }
+
+  img{
+    object-fit: cover;
+    border-radius: 3px;
+  }
+
+}
+
+.game-content, .game-content-mobile{
+  margin-left: 1rem;
+  font-size: $body-font;
+  font-size: 16px;
+  font-weight: 400;
+
+  h4{
+    font-size: 20px;
+    font-family: $headers-font;
+    margin-bottom: 1,5rem;
+  }
+
+  p{
+    margin: 0;
+  }
+  span{
+      color: gray;
+      font-weight: 300;
+    }
+
+}
+
+.game-published{
+
+  padding: 1rem 0px;
+
+  font-size : 12px ;
+}
+
+.game-card-mobile{
+  background: $lightest-gray;
+
+  @media (min-width: 768px){
+    display: none;
+  }
+
+  }
+  .game-content-mobile{
+
+    margin-top: 20px;
+    margin-bottom: 80px;
+  }
+
+  img{
+    width : 95%;
+    height: 350px;
+    margin: 15px;
+    object-fit: cover;
+    border-radius: 3px;
+  }
+  .picture-mobile{
+    height: 350px;
+  }
+
+  .title-mobile{
+    padding: 10px 0px;
+  }
+
+h4{
```

Comment:
missing space
And be careful ⚠️: Here you're setting style for each `h4` on the website.

Reply:
Yep, sorry for the mistake, I created a class .title-inside-mobile, for better scopre :-/